### PR TITLE
Debugging KeysError in enclave

### PR DESCRIPTION
**Pull request by reuvenpo**

### CODE REVIEW

File: enigma-runtime-t/src/eng_resolver.rs [hunk 1]
```diff
@@ -103,25 +103,26 @@ impl ImportResolver {
```
Comment (moriaab) [inline enigma-runtime-t/src/eng_resolver.rs]: Lovely refactoring!

File: enigma-runtime-t/src/eng_resolver.rs [hunk 2]
```diff
@@ -103,25 +103,26 @@ impl ImportResolver {
...
+            "write_eth_bridge" => (signatures::WRITE_ETH_BRIDGE, ids::WRITE_ETH_BRIDGE_FUNC),
+            "gas" => (signatures::GAS, ids::GAS_FUNC),
+            "rand" => (signatures::RAND, ids::RAND_FUNC),
+            "encrypt" => (signatures::ENCRYPT, ids::ENCRYPT_FUNC),
+            "decrypt" => (signatures::DECRYPT, ids::DECRYPT_FUNC),
             _ => return Err(wasmi::Error::Instantiation(format!("Export {} not found", field_name))),
         };
 
+        let func_ref = FuncInstance::alloc_host(signature.into(), id);
         Ok(func_ref)
```
Comment (moriaab) [inline enigma-runtime-t/src/eng_resolver.rs:126]: Lovely refactoring!

File: clone.Dockerfile [hunk 3]
```diff
@@ -0,0 +1,10 @@
+FROM alpine
```
Comment (moriaab) [inline clone.Dockerfile:1]: Please add to this file a comment with the purpose of this file.

**Review by moriaab — APPROVED** [review]

File: enigma-runtime-t/src/eng_resolver.rs (hunk 2 above)
Comment (reuvenpo) [reply enigma-runtime-t/src/eng_resolver.rs:126]: Thanks!

**Pull request merged**